frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeaderView.tsx

```diff
@@ -27,10 +27,10 @@ import { DashboardTabs } from "../../components/DashboardTabs/DashboardTabs";
 
 interface DashboardHeaderViewProps {
   editingTitle: string;
```

Comment:
There's a default value for it

```suggestion
  editingTitle?: string;
```